src/app/shared/navbar/searchbar/searchbar.ts

```diff
@@ -0,0 +1,51 @@
+import {Component, HostBinding} from '@angular/core';
+import {Router} from '@angular/router';
+import {FormControl} from '@angular/forms';
+import {MdSnackBar} from '@angular/material';
+import {Observable} from 'rxjs';
+
+import {DocumentationItems, DocItem} from '../../documentation-items/documentation-items';
+
+
+@Component({
+  selector: 'search-bar-component',
+  templateUrl: './searchbar.html',
+  styleUrls: ['./searchbar.scss']
+})
+
+export class SearchBar {
+  @HostBinding('class.expanded') _isExpanded = false;
+  public allDocItems: DocItem[];
+  public filteredSuggestions: Observable<DocItem[]>;
+  public searchControl: FormControl = new FormControl('');
+
+  constructor(
+    private _docItems: DocumentationItems,
+    private _router: Router,
+    private _snackBar: MdSnackBar
+  ) {
+    this.allDocItems = _docItems.getAllItems();
+    this.filteredSuggestions = this.searchControl.valueChanges
+      .startWith(null)
+      .map(item => this.filterSearchSuggestions(item));
+  }
+
+  public toggleIsExpanded() {
+    this._isExpanded = !this._isExpanded;
```

Comment:
The autocomplete trigger has a `panelOpen` property. Have you tried using that instead of toggling it here? You should be able to do a ViewChild query for `'auto'` to get a reference to the trigger.

```
host: {
   '[class.expanded]': '_autocompleteTrigger?.panelOpen'
}
```